Resources/Computing/ARCComputingElement.py

```diff
@@ -91,27 +108,38 @@ def submitJob( self, executableFile, proxy, numberOfJobs = 1 ):
     batchIDList = []
     stampDict = {}
 
+    # Do not need to loop over these ... the pilot proxy is the same for everyone
+    usercfg = arc.UserConfig()
+    usercfg.CredentialString(proxy)
+    endpoint = arc.Endpoint(cfgPath("gsiftp://", self.ceHost, ":2811/jobs"), arc.Endpoint.JOBSUBMIT,
```

Comment:
For that kind of path manipulation, you should use a real url manipulation library, like urlparse